src/EditorFeatures/CSharpTest/NavigateTo/NavigateToTests.cs

```diff
@@ -55,6 +55,20 @@ public async Task FindClass(TestHost testHost)
             });
         }
 
+        [Theory]
+        [CombinatorialData]
+        public async Task FindRecord(TestHost testHost)
+        {
+            await TestAsync(
+testHost, @"record Goo
+{
+}", async w =>
+            {
+                var item = (await _aggregator.GetItemsAsync("Goo")).Single(x => x.Kind != "Method");
+                VerifyNavigateToResultItem(item, "Goo", "[|Goo|]", PatternMatchKind.Exact, NavigateToItemKind.Record, Glyph.ClassInternal);
```

Comment:
The `NavigateToItemKind` used here doesn't have "Record" in it.
I think this one comes from a package reference "Microsoft.VisualStudio.Language.NavigateTo.Interfaces"

@CyrusNajmabadi What's the correct action here?
Is this package developed in this same repo or it's internal? Does it need to be updated first? Why we even need it if we have another NavigateToItemKind available?

Reply:
We woudl need to map our type to something in the existing enum.  

> Is this package developed in this same repo or it's internal? 

It's part of VS, outside of Roslyn :)

> Does it need to be updated first?

Nope.  We can just map our concepts to theirs.  We can decide later if they need to expose the concept as well.

Reply:
@CyrusNajmabadi I'm not sure what you meant by this.

Reply:
come to discord and we can discuss things there :)